index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
```

Comment:
- [Optional] Consider changing the title of the document to the title of your project so that your project title will be displayed instead of "Document" when opened in the browser.😊

Reply:
Hello @Kamba56,
It seems like changing the branch name closed this pull request. What should I do in this case, Create a new one even though it ain't the preferred option?

Reply:
If you can restore the pull request, please do and implement other changes and ask for a re-review. The next reviewer should approve it without you changing the branch name. Please do change the branch name once your project has been approved.

Reply:
Hello @DaveZag,
I apologize for the mix-up. I didn't know that changing the branch name will close the PR. Kindly try restoring the PR and if it doesn't restore, kindly create another PR and write a short comment for the code reviewer explaining what happened and your project will be approved. Thank you

Reply:
Thanks...